README.md

```diff
@@ -18,6 +18,13 @@ If bundler is not being used to manage dependencies, install the gem by executin
 
     $ gem install flatito
 
+### Nixpkgs package
+
+It is also available as [nixpkgs](https://github.com/ceritium/flatito/issues/8) thanks to @Rucadi
```

Comment:
I would put the link to: https://search.nixos.org/packages?channel=unstable&show=flatito rather than the issue probably?